src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 96px;
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
```

```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2),
........... {
  background-image: url(images/star-active.svg);
```
combine all selectors with a comma